client/lib/plans/constants.js

```diff
@@ -508,10 +522,12 @@ export const PLANS_LIST = {
 		] ),
 		getSignupFeatures: () => compact( [
 			FEATURE_OFFSITE_BACKUP_VAULTPRESS_REALTIME,
-			FEATURE_ONE_CLICK_THREAT_RESOLUTION,
+			FEATURE_VIDEO_CDN_UNLIMITIED,
+			FEATURE_SEO_PREVIEW_TOOLS,
 			FEATURE_ALL_PREMIUM_FEATURES
 		] ),
 		getBillingTimeFrame: () => i18n.translate( 'per year' ),
+		getSignupBillingTimeFrame: () => 'per year', //PLANS A/B TEST: Translate if test passes
```

Comment:
Any good reason to not wrap this one in a `translate` call immediately? This string is already translated in the above line.